documentation/notes/joss_paper/paper.md

```diff
@@ -47,11 +47,10 @@ A future development is planned that will enable the user to specify the basis o
 
 The model implementation consists of submodules to set up the model's parameters and to compute the tensor that defines the coefficients in the tendencies of the model variables; more details can be found in @DDV2016.
 This tensor is used by the code to compute the tendencies function and its Jacobian matrix. These functions can then be fed to the `qgs` built-in Runge-Kutta integrator or 
-to another integrator implemented by the user. As an example, the usage of the Julia DifferentialEquations.jl [@RN2017] integration package through the Python diffeqpy [@diffeqpy] package is provided.
-Technical details about this implementation can be found in the *Code Description* section of the included documentation.
+to another integrator implemented by the user. As an example, the usage of the Julia `DifferentialEquations.jl` [@RN2017] integration package through the Python `diffeqpy` [@diffeqpy] package is provided.
 The tangent linear and adjoint models [@K2003] are also available and allow to conduct easily data assimilation and linear sensitivity analysis experiments.
 
-The model implementation uses Numpy [@vCV2011; @O2006] and SciPy [@scipy] for arrays and computations support, as well as Numba [@numba] and sparse [@sparse] to considerably accelerate the tensor products computation used to compute the tendencies.
+The model implementation uses NumPy [@vCV2011; @O2006] and SciPy [@scipy] for arrays and computations support, as well as Numba [@numba] and sparse [@sparse] to considerably accelerate the tensor products computation used to compute the tendencies.
```

Comment:
See previous comment, but in short either 'NumPy' (general name) or `numpy` (package name, as a literal i.e monospaced).